### ISSUE TITLE
Infinite loop in exit function

### ISSUE THREAD

**Issue by elmart-devo** [yqrashawn/GokuRakuJoudo#211]
Hi, it seems I must be doing mistake when trying to use goku. But the real weird thing is that, when goku tries to exit and report an error, it seems to enter an infinite loop within the exit function, leading to a stack overflow.

```
Exception in thread "main" java.lang.StackOverflowError
	at com.oracle.svm.core.graal.snippets.StackOverflowCheckImpl.newStackOverflowError0(StackOverflowCheckImpl.java:328)
	at com.oracle.svm.core.graal.snippets.StackOverflowCheckImpl.newStackOverflowError(StackOverflowCheckImpl.java:324)
	at com.oracle.svm.core.graal.snippets.StackOverflowCheckImpl.throwNewStackOverflowError(StackOverflowCheckImpl.java:304)
	at clojure.lang.LazilyPersistentVector.createOwning(LazilyPersistentVector.java:25)
	at clojure.lang.RT.vector(RT.java:1609)
	at clojure.lang.Tuple.create(Tuple.java:26)
	at karabiner_configurator.core$exit.invokeStatic(core.clj:32)
	at karabiner_configurator.core$exit.invoke(core.clj:31)
	at karabiner_configurator.core$exit.invokeStatic(core.clj:32)
	at karabiner_configurator.core$exit.invoke(core.clj:31)
	at karabiner_configurator.core$exit.invokeStatic(core.clj:32)
	at karabiner_configurator.core$exit.invoke(core.clj:31)
	at karabiner_configurator.core$exit.invokeStatic(core.clj:32)
	at karabiner_configurator.core$exit.invoke(core.clj:31)
	at karabiner_configurator.core$exit.invokeStatic(core.clj:32)
	at karabiner_configurator.core$exit.invoke(core.clj:31)
	at karabiner_configurator.core$exit.invokeStatic(core.clj:32)
```
I'm on MacOS Ventura 13.0, with brew-installed goku 0.5.7.

**Comment by yqrashawn** [comment]
fixed in b429bfd541aa1277d4b95bb51b118c39a046e0c6, not published yet

**Comment by Bargsteen** [comment]
Could you publish this fix? Goku is not usable for me due to this bug.

**Comment by yqrashawn** [comment]
This is finally published in v0.7.0
